jerry-core/jerry.c

```diff
@@ -2149,6 +2152,31 @@ jerry_is_valid_cesu8_string (const jerry_char_t *cesu8_buf_p, /**< CESU-8 string
 } /* jerry_is_valid_cesu8_string */
 
 /**
+ * If JERRY_VM_EXEC_STOP is defined the periodically called callback
+ */
+void
+jerry_set_vm_exec_stop_callback (jerry_vm_exec_stop_callback_t stop_cb, /**< periodically called user function */
+                                 void *user_p, /**< pointer passed to the function */
+                                 uint32_t frequency) /**< frequency of the function call */
```

Comment:
This is described in the documentation. There is no real unit. If the callback could be called `N` times, it is actually called `floor(N/frequency)` times